Wikipedia/UI-V5/WMFHomeSectionSchema.m

```diff
@@ -122,7 +128,7 @@ - (void)update:(BOOL)force {
     //Check Tweak
     if (!FBTweakValue(@"Home", @"General", @"Always update on launch", NO)) {
         //Check force flag and minimum relaod time
-        if (!force && [[NSDate date] timeIntervalSinceDate:self.lastUpdatedAt] < WMFHomeMinimumAutomaticReloadTime) {
+        if (!force && self.lastUpdatedAt && [[NSDate date] timeIntervalSinceDate:self.lastUpdatedAt] < WMFHomeMinimumAutomaticReloadTime) {
```

Comment:
maybe a product question, but should we make sure "featured" is refreshed when it's a new day?